### ISSUE TITLE
Figure out how to record the arch call

### ISSUE THREAD

**Issue by ralphbean** [konflux-ci/community#1]
We want to figure out how to record the weekly arch call. When I look at the [google meet requirements](https://support.google.com/meet/answer/9308681?hl=en#zippy=%2Clearn-how-to-record-a-meeting%2Crequirements-to-record-a-video-meeting) it looks like we need a subscription in order to record.

We don't have a subscription. Any suggestions on a way forward?

**Comment by ralphbean** [comment]
Some foss options are jitsi and "big blue button". A friend reported that BBB's recording works, even if the interface is a little clunky.

**Comment by ralphbean** [comment]
I don't think I can take this forwards alone. If you have opinions on which new video option to try and some time to volunteer to help set it up, please speak up here or reach out to me and let's see if we can make some progress.

**Comment by kasemAlem** [comment]
@ralphbean , before we could record all sessions . what changed ? aren't we enterprise accounts anymore ?
 what I saw that we can use 3rd party Chrome extensions  I'll make some learning on those  then share my insights

**Comment by ralphbean** [comment]
@kasemAlem our old internal Red Hat architecture call was on our RH google account. When we shifted to open up the call, we thought it best to _not_ host it on Red Hat's account, in an effort to reduce barriers for new people to join. We're not (currently) hosting the meeting on Red Hat's account.

**Comment by kasemAlem** [comment]
I 'found that a free extension [Screen Recorder](https://workspace.google.com/marketplace/app/screen_recorder_voice_recorder_video_rec/659492711995?pann=ogb) is free of charge, but it's not allowed by administrator, is this something we can ask admins enable ?  also it's local, 

other tool is  [loom](https://www.loom.com/looms/videos)  extension ,  I've tried it for recording a minute looks good 
my concern is that it upload the videos on their website via workspace that each user should be signed in and joined a workspace

**Comment by kasemAlem** [comment]
third one that I think it can be a good candidate which is also chrome extension and it saves the recording locally 
[screen.io](https://chromewebstore.google.com/detail/screen-recorder/hniebljpgcogalllopnjokppmgbhaden) , also it's free

**Comment by happybhati** [comment]
I was also trying suggested tools and some extensions. However, I came to the conclusion that recordings will be saved locally because no one will provide free cloud space for storing them. There was an option to make the Google Meet live stream on YouTube and store the video/livestream on YouTube, but that will also require a subscription. I did not find any better option that is free and does not require manual adjustments, like recording locally and posting on some platforms.

We can ask maybe Burr Sutter, he must have done something similar. I guess live streaming from a YouTube channel named Konflux can be an option.

**Comment by jflowers** [comment]
I think this would work:

Create a YouTube channel with a name other than you from your Google(Red Hat) account: https://support.google.com/youtube/answer/1646861?hl=en#:~:text=Create%20a%20channel%20with%20a%20business%20or%20other%20name

Create a shared Google Calendar and Google Group from your personal Google account.

Create a calendar invite from your Red Hat account and invite both the Google Group and the calendar id from the shared Google Calendar. Make sure that the meeting access type is set to Open by clicking the gear in the Google Meet section of the invite.

Record the meeting.

When the recording is available upload it to YouTube.

**Comment by ralphbean** [comment]
A thought from the community call today: perhaps use of Red Hat's enterprise google account is not as much of a turn off as we worried that it would be. I've joined a few other community's meetings over the last week - one was sponsored by github and they used github's corporate zoom account. It was fine - and seemed practical.

Now that we've struggled with this for a little over a month - is there any strong objection out there to using Red Hat's corporate google account for our meeting?

I would expect that if a SIG or any other meeting in our community would be perfectly fine to use the hosting details of whoever is running the meeting, whether they work at RH or another company.
